data/transaction/status.go

```diff
@@ -23,12 +23,13 @@ const (
 
 // StatusComputer computes a transaction status
 type StatusComputer struct {
-	MiniblockType    block.Type
-	SourceShard      uint32
-	DestinationShard uint32
-	Receiver         []byte
-	TransactionData  []byte
-	SelfShard        uint32
+	MiniblockType             block.Type
+	IsMiniblockFullyNotarized bool
```

Comment:
IsMiniblockFinalized ?

Reply:
@sasurobert Isn't the block finalized only at n+1 ?

Reply:
IsMiniBlockFully notarized is for when even the destination processes the miniblock and the result of the process is finalized, included on meta, and meta is finalized.

Reply:
Fixed.